tripgo.swagger.yaml

```diff
@@ -2506,6 +2723,73 @@ definitions:
       urls:
         - nuremberg-bv-de.hadron.buzzhives.com
 
+  RealtimeSnapshot:
+    description: Real Time Snapshot stats.
```

Comment:
OK, that way in a much simpler implementation too and saves lot of detailed cached info. Should it be an object with 3 booleans?